src/bp/admin/ui/src/health/monitoring/index.tsx

```diff
@@ -272,38 +271,36 @@ class Monitoring extends Component<Props, State> {
     }
 
     return (
-      <div className="monitoring">
-        <Row>
-          <Col md={6}>
+      <div>
+        <div className={style.row}>
+          <div className={style.col}>
```

Comment:
Not a fan of the naming here but I know this was just to make it work. A Front End ninja can fix this later on I guess

Reply:
It was just to get rid of bootstrap, not sure how else I could name it, i'll wait for a frontend ninja